### ISSUE TITLE
OSX Error The language service could not be started

### ISSUE THREAD

**Issue by sunitjoshi** [PowerShell/vscode-powershell#340]
**Please fill in these details so that we can help you!**

### System Details

- Operating system name and version: Darwin Kernel Version 16.1.0 (macOS Sierra 10.12.1)
- VS Code version: 1.7.1
- PowerShell extension version: 0.7.2
- Output from `$PSVersionTable`:
Name                           Value                                                        
----                           -----                                                        
PSVersion                      6.0.0-alpha                                                  
PSEdition                      Core                                                         
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                      
BuildVersion                   3.0.0.0                                                      
GitCommitId                    v6.0.0-alpha.10                                              
CLRVersion                                                                                  
WSManStackVersion              3.0                                                          
PSRemotingProtocolVersion      2.3                                                          
SerializationVersion           1.1.0.1      
```
Evaluate $PSVersionTable in PowerShell and paste the output here
```

### Issue Description
Unable to start PowerShell from VS Code. Getting an error "Language service could not be started: Error: spawn EACCESS"

I'm able to start PowerShell from terminal with no issues. powerShellExePath is set to /usr/local/bin which is symlinked to /usr/local/microsoft/powershell/6.0.0-alpha.10/powershell

### Attached Logs
The log files have 0 bytes even though I've set the user-settings!

Follow the instructions in the [README](https://github.com/PowerShell/vscode-powershell#reporting-problems)
about capturing and sending logs.

**Comment by daviwil** [comment]
Hey @sunitjoshi, do you have OpenSSL installed?  Check out these instructions:

https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#1-powershell-intellisense-does-not-work-cant-debug-scripts

Can you confirm which version of OpenSSL you have installed?

**Comment by sunitjoshi** [comment]
The version is:
OpenSSL 0.9.8zh 14 Jan 2016

**Comment by sunitjoshi** [comment]
One more thing, I've these outputs:
○ → **which** openssl
/usr/bin/openssl

○ → ln -s /usr/local/Cellar/openssl/1.0.2j/lib/libssl.1.0.0.dylib /usr/local/lib/libssl.1.0.0.dylib 
ln: /usr/local/lib/libssl.1.0.0.dylib: **File exists**

**Comment by daviwil** [comment]
Hi @sunitjoshi, this issue should be fixed in the latest 0.8.0 update, please try it out and let me know if it works for you.  I'll reopen the issue if you still see problems after the update.
